csharp/ql/test/library-tests/diagnostics/DiagnosticExtractorErrors.expected

```diff
@@ -0,0 +1,2 @@
+| Program.cs:9:32:9:46 | Unable to resolve target for call. (Compilation error?) | Unexpected C# extractor error in Program.cs: Unable to resolve target for call. (Compilation error?) | 3 |
+| file://:0:0:0:0 | Unhandled literal type | Unexpected C# extractor error: Unhandled literal type | 3 |
```

Comment:
This change revealed this extraction error, which is caused by an earlier change introducing default argument values. I only handled c# constant values there, but we're also extracting dlls produced from other languages such as vb.net, where you can define `object` constant value as in https://docs.microsoft.com/en-us/dotnet/api/microsoft.visualbasic.filesystem.fileput?view=net-5.0#Microsoft_VisualBasic_FileSystem_FilePut_System_Object_System_Object_System_Object_ where the third parameter is optional.

Reply:
The fix is coming in https://github.com/github/codeql/pull/5745